### ISSUE TITLE
text alignment incorrect for .navbar-fixed-top .navbar-right

### ISSUE THREAD

**Issue by sabrinaluo** [twbs/bootstrap#19997]
Hi, I'm using Chrome Version 50.0.2661.102 (64-bit)
the navbar looks like below:
![selection_095](https://cloud.githubusercontent.com/assets/5300359/15667376/871bfd1c-2748-11e6-8f49-cac8f3e19f05.png)

source code can be found here http://codepen.io/hiitea/pen/vKEBVO

I found that with adding below code into css, it looks correct
```
#nav-demo{
  padding-right:15px;
}
```
![selection_096](https://cloud.githubusercontent.com/assets/5300359/15667428/c957eb28-2748-11e6-8fe4-60b2228993a6.png)

It seems `.navbar-static-top` and `.navbar-fixed-bottom` have the same issue also.

**Comment by twbs-lmvtfy** [comment]
Hi @sabrinaluo!

You appear to have posted a live example (http://codepen.io/hiitea/pen/vKEBVO.html), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 8, column 1 thru column 6: Stray start tag ```` html ````.
* line 10, column 1 thru column 6: Stray start tag ```` head ````.
* line 11, column 3 thru column 94: Bad value ```` ////maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css ```` for attribute ```` href ```` on element ```` link ````: Unexpected slash or backslash.
* line 11, column 3 thru column 94: Element ```` link ```` is missing required attribute ```` property ````.
* line 12, column 1 thru column 7: Stray end tag ```` head ````.
* line 14, column 1 thru column 6: Start tag ```` body ```` seen but an element of the same type was already open.
* line 33, column 1 thru column 7: End tag for  ```` body ```` seen, but there were unclosed elements.
* line 15, column 3 thru column 7: Unclosed element ```` div ````.
* line 36, column 1 thru column 7: Stray end tag ```` body ````.
* line 36, column 1 thru column 7: End tag for  ```` body ```` seen, but there were unclosed elements.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Cleaning that up slightly: http://codepen.io/cvrebert/pen/yJyBZj

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://codepen.io/cvrebert/pen/yJyBZj.html), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 8, column 1 thru column 6: Stray start tag ```` html ````.
* line 10, column 1 thru column 6: Stray start tag ```` head ````.
* line 11, column 3 thru column 102: Element ```` link ```` is missing required attribute ```` property ````.
* line 12, column 1 thru column 7: Stray end tag ```` head ````.
* line 14, column 1 thru column 6: Start tag ```` body ```` seen but an element of the same type was already open.
* line 33, column 1 thru column 7: End tag for  ```` body ```` seen, but there were unclosed elements.
* line 15, column 3 thru column 7: Unclosed element ```` div ````.
* line 36, column 1 thru column 7: Stray end tag ```` body ````.
* line 36, column 1 thru column 7: End tag for  ```` body ```` seen, but there were unclosed elements.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Duplicate of #18355.
